src/snap/actions/pairing.ts

```diff
@@ -134,7 +140,16 @@ export const getToken = async (currentAccountAddress?: string) => {
 		if (currentAccountAddress && tempAccountAddress == null) {
```

Comment:
Our getToken function is quite complex and lose its single responsibility. We should split getToken, pairing into smaller functions. We also need to update the name of `getToken`, getToken is not the only action of this function.